control/injector/Injector.php

```diff
@@ -594,6 +594,20 @@ public function inject($object, $asType=null) {
 			}
 		}
 
+		// Populate named methods
+		if (isset($this->specs[$objtype]['methods'])) {
```

Comment:
I don't think that's true; I just tested this code and it doesn't error:

    <?php
    $from = array();
    echo isset($from['bad']['to']['worse']) ? "yes\n" : "no\n";

...at least not on PHP 5.4. Has it started erroring on a more recent version?